diff --git a/eng/Versions.props b/eng/Versions.props
index 57159037..424f9475 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -10,7 +10,7 @@
       needed to raise the version number to preserve continuity. In preview9 we should remove these
       to conform to Arcade defaults. https://github.com/dotnet/core-setup/issues/7348
     -->
-    <PreReleaseVersionLabel>preview8</PreReleaseVersionLabel>
+    <PreReleaseVersionLabel>preview8-criteo2</PreReleaseVersionLabel>
     <!-- Enable to remove prerelease label. -->
     <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
     <DotNetFinalVersionKind Condition="'$(StabilizePackageVersion)' == 'true'">release</DotNetFinalVersionKind>
